### PR TITLE
[BPS][Analytics]Move BPS Analytics Dashboards and UDF to EI

### PR DESCRIPTION
**Pull request by isurusuranga**

### CODE REVIEW

File: distribution/src/analytics/conf/spark-udf-config.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (milindaperera) [inline distribution/src/analytics/conf/spark-udf-config.xml]: Year 2017

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/Eventreceiver_bpmn_process.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (milindaperera) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/Eventreceiver_bpmn_process.xml]: License header

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/artifact.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (milindaperera) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/artifact.xml]: License header

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_service_task_1.0.0/Eventreceiver_bpmn_service_task.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (milindaperera) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_service_task_1.0.0/Eventreceiver_bpmn_service_task.xml]: License header

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_task_1.0.0/Eventreceiver_bpmn_task.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (milindaperera) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_task_1.0.0/Eventreceiver_bpmn_task.xml]: add License headers for the rest of files also

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/Eventreceiver_bpmn_process.xml (hunk 2 above)
Comment (isurusuranga) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/Eventreceiver_bpmn_process.xml]: fixed.
https://github.com/wso2/product-ei/pull/344/commits/6cb0a26ca6da3c8ec3f7949266f7f8349d8d89a2

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/artifact.xml (hunk 3 above)
Comment (isurusuranga) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_process_1.0.0/artifact.xml]: fixed.
https://github.com/wso2/product-ei/pull/344/commits/6cb0a26ca6da3c8ec3f7949266f7f8349d8d89a2

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_service_task_1.0.0/Eventreceiver_bpmn_service_task.xml (hunk 4 above)
Comment (isurusuranga) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_service_task_1.0.0/Eventreceiver_bpmn_service_task.xml]: fixed.
https://github.com/wso2/product-ei/pull/344/commits/6cb0a26ca6da3c8ec3f7949266f7f8349d8d89a2

File: features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_task_1.0.0/Eventreceiver_bpmn_task.xml (hunk 5 above)
Comment (isurusuranga) [inline features/org.wso2.ei.analytics.feature/src/main/capp/bpmn/common/Eventreceiver_bpmn_task_1.0.0/Eventreceiver_bpmn_task.xml]: fixed.
https://github.com/wso2/product-ei/pull/344/commits/6cb0a26ca6da3c8ec3f7949266f7f8349d8d89a2

**Comment by nandika** [comment]
Change the packages from org.wso2.ei.bps.... to org.wso2.ei.bpmn... 

Also renamed the capp from org.wso2.bpmn.analytics-1.0.0.car to org.wso2.ei.bpmn.analytics-1.0.0.car

**Pull request merged**